Tools/cases_generator/generate_cases.py

```diff
@@ -73,6 +74,34 @@ def block(self, head: str):
             yield
         self.emit("}")
 
+    def stack_adjust(self, diff: int):
+        if diff > 0:
+            self.emit(f"STACK_GROW({diff});")
+        elif diff < 0:
+            self.emit(f"STACK_SHRINK({-diff});")
+
+    def declare(self, dst: StackEffect, src: StackEffect | None):
+        if dst.name == UNUSED:
+            return
+        type = f"{dst.type} " if dst.type else "PyObject *"
```

Comment:
Should we just set the default value of `type` to `"PyObject *"` in the parser to avoid this move here and elsewhere?

Reply:
Then we'd be generating slightly un-idiomatic code where there's a space after the star. I'm not sure what bugs me more. I think the code I already wrote wins though. :-)